pyunfold/unfold.py

```diff
@@ -121,6 +121,8 @@ def iterative_unfold(data=None, data_err=None, response=None,
     for name in inputs:
         if inputs[name] is None:
             raise ValueError('The input for "{}" must not be None.'.format(name))
+        if np.any(np.array(inputs[name]) < 0.):
```

Comment:
Could you use `np.asarray` instead of `np.array`? That way, if the input data are already `ndarray`s, then they aren't copied.

Reply:
Just wondering, is there a reason to use `0.` here instead of just `0`?

Reply:
I used `0.` to ensure floating point comparison. I used `np.array()` because it merely copies and the `cast_to_array` method is used just after, so I thought it would be redundant. Either way it should be fine of course :)